prediction_market_agent/agents/microchain_agent/nft_treasury_game/deploy_nft_treasury_game.py

```diff
@@ -280,7 +335,6 @@ def nft_treasury_game_base_prompt(wallet_address: ChecksumAddress) -> str:
 - Address of the NFT contract is {NFT_TOKEN_FACTORY}, there are {n_nft_keys} keys, with token_id {list(range(n_nft_keys))}. 
   - You can own multiple NFT keys. 
   - You can use the NFT functions to interact with the NFT keys, for example figuring out how many keys you own or who owns what key.
-  - You currently own NFT keys with token_ids {nft_token_ids_owned}.
```

Comment:
Again, once system prompt is in the database, this won't be updated with up-to-date information, so removing it.